### PR TITLE
Re-applying the Jit indexing build path

### PR DESCRIPTION
**Pull request by kushanam**
Re-submitting the build path for JIT 64 bit indexing (rolling back the b6569d7) and fixing the name length (to fix the the Winodws build issue).
Attn: @frgossen

### CODE REVIEW

File: tensorflow/core/kernels/mlir_generated/build_defs.bzl [hunk 1]
```diff
@@ -89,16 +89,18 @@ _gen_mlir_op_rule = rule(
     implementation = _gen_mlir_op_impl,
 )
 
-def _gen_mlir_op(op, type, platform, output_type):
+def _gen_mlir_op(op, name, type, platform, output_type):
     _gen_mlir_op_rule(
         name = "generate_{op}_{platform}_{type}_{output_type}_mlir".format(
```
Comment (frgossen) [inline tensorflow/core/kernels/mlir_generated/build_defs.bzl:94]: Is this uique w/o the name in it? If the name is not needed, why is it passed below?

**Review by frgossen — COMMENTED** [review]
Thanks! Did this pass the tests for you?

File: tensorflow/core/kernels/mlir_generated/build_defs.bzl (hunk 1 above)
Comment (kushanam) [reply tensorflow/core/kernels/mlir_generated/build_defs.bzl]: name is removed from function

File: third_party/gpus/find_cuda_config.py.gz.base64__ [hunk 2]
```diff
@@ -0,0 +1 @@
...tRo3HeAyZ4Jqh4aq11e69ViP2cCzzgQo6mqH5sSMh5LDR7HB5TaV2uFbMHPC0s4JjnPzkoVB2x0KrXci0AmVMgMsW/KkEJ4V+cO3k29/X50k2WJ3dLRhkYtVnC2LZC1XTGlfTPP2nB5TN5ewYzoJzMrbDO/mgAZKUtTV1kBexnEXy3uJ8KIPdGvO6Pdr53w3jVfz9Obf/N2rv+2y8Xb/HTF2ffeu4mr3T/8BhtM2uhuPc17V98JZh7YeUtt+dUImn/Mk/oc05CpWWYpgmsiduAN5IZaWUpHw4lZeUKSuBlAarVyGtrRc8KTqotd1YkFBKZBFwkPhe9fXdJzMVnBsoHhobK1tanIUh3TFSU53dG26aUmpi2ShutXlgF3xfLYKIHCBq5eHVfEFbZBcFwH+vtofjse9xiZR53if51yCgpwRY7XzRs74ebVLdx4MJo86F9nokzZ01nbiW59htPRIinugrZDjOJa+XR1F/ee/5LjSPh46OmkkaJ2b7OKxcaeLVztqpPwgnSBdLiJwrX5zV3XVgTOzoGpnZ2wkuqv6Xq+xaklr/prYihBtkHS8wXdPpNuYrGjaZ39k/v7egO2bFOYN+76xA0TtvbjjhbpyQu67MKe1cAeGtZFQX13g0FoJwebkAY20r9gxOukKvEpaW/Zc2VS4Gze6RkMtpf6sUgY57hkpN7poCKexaN0qnmpbwtMEVN/0UaemuazdJSZaiHpNIdU6aIjIWeRrFZBern2aeOxFcJsKdwmwSzA4u/+acrHxN8RSXzJplYpa99oolJaVQqtjZxWl05DkvM2rGlK9i6YhuTNk7YZkZj2faEi1OeU6Nc05tDYxqcCUph2RKU2r0OQmzF4zr22JTu76QHvor+4le1AlrJUni0B3mcBwRxMvLczRZEg3b438vIU1Z7KohTP78rQHGLMnCCvanBkjw5Uu4do4M+VdN3etpUYLhy3FcQuX7v1uD3DqTiLV6W2plYlrNM8vA33FDWGNSzE3STJI5QtQFslrejyqaekaktpb7MKzZkwo1fkyRiE5Wf/KLtHVDVUyt57zOJW9mptDiDixHuDhvKWwzrs383BNtSGCrvb0vbfFUO7QZ36FSZ7zlZeMWDv+MeVR0/2YaMsrQQO87lb4RZn7As8sbl2nW33TBhLUEo8G9XpYY1DhM5nQDWSTCTI0mXiIUPLW+zetrxKs
```
Comment (frgossen) [inline third_party/gpus/find_cuda_config.py.gz.base64__:1]: Is this change intended?

**Review by frgossen — CHANGES_REQUESTED** [review]

**Review by frgossen — APPROVED** [review]
Thanks!
